static/js/settings_bots.js

```diff
@@ -56,6 +56,14 @@ exports.generate_zuliprc_content = function (email, api_key) {
            "\n";
 };
 
+exports.generate_zuliprc_content_for_single_file = function (email, api_key) {
+    return "[" + email.substring(0, email.indexOf("-bot@zulip.localhost")) + "]" +
```

Comment:
Also, would this code be more readable with a separate `bot_name_from_email` function?